benchmarks/routines/attention.py

```diff
@@ -1150,6 +1174,11 @@ def testBatchPrefillWithRaggedKVCacheWrapper(args):
         ]:
             print("[INFO] CUTLASS backend does not support FP8. Skipping.")
             remove_cutlass = True
+        if not (
+            (head_dim_qk == 128 and head_dim_qk == head_dim_vo) or head_dim_qk == 192
+        ):
+            print("[INFO] CUTLASS backend requires head dimension to be 128 or 192")
```

Comment:
We can add other head dimensions in cutlass backends as well but thanks for adding this check here!